config/regex.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
     JOB_NAME: /^[\w-]+$/,
     // PR JOB Name can only be PR-1 or PR-1:main, group1: PR-prNum, group2: jobName
     PR_JOB_NAME: /^(PR-\d+)(?::([\w-]+))?$/,
-    // Can be ~pr, ~commit, or ~commit:branchName
-    TRIGGER: /^~(pr|commit(:.+)?)$/,
+    // Can be ~pr, ~commit, or ~commit:branchName, or ~sd@123:component
+    TRIGGER: /^~(sd@\d+:[\w-]+|pr|commit(:.+)?)$/,
```

Comment:
Can we also have a separate regex for  `~sd@\d+:[\w-]+` with capture groups? so that later we can use it to check if the trigger is external pipeline and get right groups we want out of it easily. e.g. pipelineId, jobName

Reply:
sure 👍